controllers/admin/AdminRequestSqlController.php

```diff
@@ -127,10 +127,10 @@ public function renderList()
         $this->displayInformation('
 		<strong>' . $this->trans('How do I create a new SQL query?', [], 'Admin.Advparameters.Help') . '</strong><br />
 		<ul>
-			<li>' . $this->trans('Click "Add New".', [], 'Admin.Advparameters.Help') . '</li>
-			<li>' . $this->trans('Fill in the fields and click "Save".', [], 'Admin.Advparameters.Help') . '</li>
-			<li>' . $this->trans('You can then view the query results by clicking on the Edit action in the dropdown menu', [], 'Admin.Advparameters.Help') . ' <i class="icon-pencil"></i></li>
-			<li>' . $this->trans('You can also export the query results as a CSV file by clicking on the Export button', [], 'Admin.Advparameters.Help') . ' <i class="icon-cloud-upload"></i></li>
+			<li>' . $this->trans('Click "%add_new_label%".', ['%add_new_label%' => $this->trans('Add new SQL query', [], 'Admin.Advparameters.Feature')], 'Admin.Advparameters.Help') . '</li>
+			<li>' . $this->trans('Fill in the fields and click "%save_label%".', ['%save_label%' => $this->trans('Save', [], 'Admin.Actions')], 'Admin.Advparameters.Help') . '</li>
+			<li>' . $this->trans('You can then view the query results by clicking on the "%view_label%" action in the dropdown menu', ['%view_label%' => $this->trans('View', [], 'Admin.Global')], 'Admin.Advparameters.Help') . ' <i class="icon-pencil"></i></li>
```

Comment:
No edit action?

Reply:
There is an edit action, but it only allows to edit the SQL query, no to view it To see the query results the user must click on `View`

Reply:
Ahhh, inherited strings... thanks!
🙄